sc/src/test/java/uk/ac/ebi/atlas/download/ExperimentFileLocationServiceIT.java

```diff
@@ -0,0 +1,54 @@
+package uk.ac.ebi.atlas.download;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import org.springframework.test.context.web.WebAppConfiguration;
+import uk.ac.ebi.atlas.resource.DataFileHub;
+
+import javax.inject.Inject;
+import java.io.File;
+import java.nio.file.Path;
+
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@WebAppConfiguration
+@ContextConfiguration(locations = {"classpath:applicationContext.xml", "classpath:dispatcher-servlet.xml"})
+public class ExperimentFileLocationServiceIT {
+
+    private final String EXPERIMENT_ACCESSION = "TEST-SC";
+    private final String EXPERIMENT_DESIGN_FILE_NAME = "ExpDesign-TEST-SC.tsv";
+
```

Comment:
I’ll add a *Homo sapiens* experiment from `/nfs/production3/ma/home/nf/EA/sc/all.done.txt` to https://github.com/ebi-gene-expression-group/scxa-test-experiments, we’ll add it to the SC portal and use that one to test.

Also, when adding new tests check the coverage. In this case `ExperimentFileLocationService` has 66% and it’s simple enough to go up to 100%.

Reply:
Coverage for ExperimentFileLocationService has improved in the later branch.